.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,55 @@
+name: prepare_release
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  check:
+    name: check release version
+    runs-on: ubuntu-latest
+    outputs:
+      skip: ${{ steps.bumpr.outputs.skip }}
+      version: ${{ steps.bumpr.outputs.next_version }}
+    steps:
+      - uses: actions/checkout@v2
+      - id: bumpr
+        uses: haya14busa/action-bumpr@v1
+        with:
+          dry_run: true
+  prepare:
+    name: prepare release
+    needs: check
+    if: '!needs.check.outputs.skip'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: false
+          fetch-depth: 0
+      - name: setup node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '14'
+      - name: update changelog
+        run: |
+          export NEXT_VERSION_V=${{ needs.check.outputs.version }}
+          export NEXT_VERSION=${NEXT_VERSION_V:1}
+          sed -i "/\[Unreleased\]/a ## [$NEXT_VERSION] newDateHere" CHANGELOG.md
```

Comment:
Wouldn't it make sense to use another env variable instead of `newDateHere`?

Reply:
Maybe. But I found the code to replace a string with a date on stackoverflow and used that. I'm not really good with command line stuff in general